packages/salesforcedx-vscode-core/package.json

```diff
@@ -181,6 +181,10 @@
         {
           "command": "sfdx.force.debugger.stop",
           "when": "sfdx:project_opened"
+        },
+        {
+          "command": "sfdx.force.config.list",
+          "when": "sfdx:project_opened"
         }
```

Comment:
We should add alias:list and org:display here as well. Otherwise, those commands can be executed without an sfdx folder and it'll just error out.